modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/ProductSubscriptionConfigurationUtil.java

```diff
@@ -49,7 +50,11 @@ public static CPDefinition updateCPDefinitionSubscriptionInfo(
 			GetterUtil.get(
 				productSubscriptionConfiguration.getLength(),
 				cpDefinition.getSubscriptionLength()),
-			subscriptionTypeValue, null,
+			subscriptionTypeValue,
+			UnicodePropertiesBuilder.create(
+				productSubscriptionConfiguration.getSubscriptionTypeSettings(),
```

Comment:
what if this is null?

Reply:
the method UnicodePropertiesBuilder.create check that the map is not null, if the map is null it return a Collections. <String, String> emptyMap()

Reply:
thank you!